tests/samples/unordered_imports.py

```diff
@@ -0,0 +1,22 @@
+#!/bin/env python
```

Comment:
```suggestion
```

Reply:
This comment is there to catch the case where comments from the "header" of a file should not be attached to an imports and thus should not be moved.